frontend/src/views/tournaments.js

```diff
@@ -85,15 +85,39 @@ class Tournaments extends Component {
                                     <h4 className="title">Tournament Results</h4>
                                 </div>
                                 <div className="content">
-                                    <p>Congratulations to the winners!</p>
+                                    <p>View tournament brackets here:</p>
                                     <ul>
-                                        <li>
-                                            <b>Sprint Tournament 1 <a href="https://challonge.com/rajiqb0b">(See bracket here)</a> :</b> Super Cow Powers (tscmoo)
-                                        </li>
-                                        <li>
-                                            <b>Sprint Tournament 2 <a href="https://challonge.com/6b6fjvnq">(See bracket here)</a> :</b> Super Cow Powers (tscmoo)
-                                        </li>
+                                        <li><a href="https://challonge.com/rajiqb0b">Sprint Tournament 1</a></li>
+                                        <li><a href="https://challonge.com/6b6fjvnq">Sprint Tournament 2</a></li>
```

Comment:
do we not want to publish the sprint tour winners (well, winner) at all? even if its not a prizewinner

Reply:
Good catch! Sprint tour is a prizewinner too, add them in!